changelogs/fragments/641-copy-loadlib-member.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- zos_copy - Copy failed from a loadlib member to another loadlib member. Fix now looks for error in stdout in the if statement to use -X option.
+(https://github.com/ansible-collections/ibm_zos_core/pull/641)
```

Comment:
Shouldn't this be pull/736?
Also, github is flagging the fact that there is no newline at the end of the file.

Reply:
No, as I mentioned to Demetri "in this PR I removed 601 fragment and added the correct PR # 641 for the fix in the 1.5.0 branch.", there is no change log fragment for this PR (736) bc of the triviality of the test case.

About the no newline, this is the first time I get feedback about it. I don't know if is an issue when building the change log but I'll change it.

Reply:
Marking this as resolved as it appears to be given the reviewer has passed the PR